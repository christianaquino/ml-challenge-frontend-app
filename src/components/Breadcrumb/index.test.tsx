import { render } from '@testing-library/react';
import BreadCrumb from '.';

describe('BreadCrumb component', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<BreadCrumb categories={['cat1', 'cat2', 'cat3']} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
