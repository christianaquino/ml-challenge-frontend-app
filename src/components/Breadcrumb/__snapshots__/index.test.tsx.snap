// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadCrumb component renders correctly 1`] = `
<DocumentFragment>
  <nav
    class="breadcrumb"
  >
    <ol
      class="breadcrumb--items"
    >
      <li
        class="breadcrumb--item"
      >
        cat1
      </li>
      <li
        class="breadcrumb--item"
      >
        cat2
      </li>
      <li
        class="breadcrumb--item"
      >
        cat3
      </li>
    </ol>
  </nav>
</DocumentFragment>
`;
